build.gradle

```diff
@@ -92,6 +92,7 @@ tasks.processResources {
 archivesBaseName = "wikipedia"
 josm {
   manifest {
+    oldVersionDownloadLink 14149, "v1.1.3", new URL("https://josm.gitlab.io/plugin/wikipedia/dist/v1.1.1/wikipedia.jar")
```

Comment:
I missed this line in my commit 92b0654. The URL should be https://josm.gitlab.io/plugin/wikipedia/dist/v1.1.3/wikipedia.jar (not v1.1.1), right?

Reply:
Yep. I saw 1.1.1 originally (in releases), and used that, then went back to double-check tags, saw 1.1.3, and only changed one of them.